### ISSUE TITLE
Static audio at first launch after boot, and distorted audio cue (Pulseaudio)

### ISSUE THREAD

**Issue by Brottweiler** [mumble-voip/mumble#2350]
# Information
**OS**: `Linux 4.5.7-1-ck x86_64 GNU/Linux` (Archlinux)
**Sound server**: Pulseaudio `8.0`
**Mumble version**: `1.3.0_1324_g04b2635`
**Pulse `daemon.conf` (user)**: https://gist.github.com/77291a3ec82fe4066f6a150ad68d2fd3
**Pulse `daemon.conf` (system)**: https://gist.github.com/7608fdf9d2cfedcb06c01980619e2a61 (at some point I had edited this, it should be default now, though)

# Issue

## Static sound
Most of the time after I've just started my computer, I open mumble and all audio has static in it. I would notice it if there was already audio playing, like music, or if I connect to a server. The audio has static, until I restart pulseaudio with `pulseaudio -k`. After that, every mumble launch is fine.

So basically, after every boot after I login my system, I would have to launch and quit mumble, then restart pulse. Restarting pulse *before* mumble makes no difference, mumble has to be started first, then restart pulse.

This issue happens with both the stable `1.2.16` version, the `1.3.0` snapshots, aswell as the latest git.

# Audio Cue distortion
Also, if I enable Audio Cue, which plays a sound when I press and release PTT, the audio sounds distorted. Basically, it sounds like the audio cuts out before it finishes playing.

**Comment by Kissaki** [comment]
> Also, if I enable Audio Cue, which plays a sound when I press and release PTT, the audio sounds distorted. Basically, it sounds like the audio cuts out before it finishes playing.

And that is the case even after you fixing other sound with `pulseaudio -k`?

**Comment by Brottweiler** [comment]
Yes.

**Comment by Lartza** [comment]
This started happening to me on GNOME, it did not happen on an another DE (Budgie)

**Comment by Brottweiler** [comment]
I think I've solved the issue with all sound going static when launching mumble. My text to speech wasn't working, and I was trying to solve it. I found some thread about pulseaudio and distortion, and someone mentioned speech-dispatcher.

I decided to try and build mumble with `no-speechd`, and so far, 3 boots, no static.

My guess is that when speech-dispatcher starts, after pulse starts, it goes static. Restarting pulse, speech-dispatcher is already running, so that fixes it. This is probably why restarting pulse after boot, before starting mumble, doesn't make a difference.

**Comment by Brottweiler** [comment]
Alright, I've confirmed it. The "static audio" issue when starting mumble is unrelated to mumble. Starting `speech-dispatcher` manually from terminal causes it, killing it stops the static. It's literally caused by speech dispatcher, not mumble.

The other is still an issue, I'll make a new issue about it.

To fix the static audio when starting mumble, I just have to build mumble without speech-dispatcher.

**Comment by G2G2G2G** [comment]
Any fix for this yet or do you need to run a @reboot speech-dispatcher && pulseaudio -k  in a crontab?
